Jenkinsfile

```diff
@@ -41,6 +41,7 @@ pipeline {
                 beforeAgent true
                 allOf {
                     branch 'master';
+                    branch 'release/*';
```

Comment:
This is the "heart" of this PR, it activates the creation of docker releases for the branches `release/*`, which where ignored by the `Release Branch Release to Unofficial Asset Stores` jenkins task before.